pkg/services/system_intakes.go

```diff
@@ -109,11 +135,32 @@ func NewFetchSystemIntakeByID(
 			logger.Error("failed to fetch system intake")
 			return &models.SystemIntake{}, &apperrors.QueryError{
 				Err:       err,
-				Model:     "system intake",
+				Model:     "System intake",
 				Operation: apperrors.QueryFetch,
 			}
 		}
 		return intake, nil
 	}
 
 }
+
+// NewValidateAndSubmitSystemIntake is a service to submit system intake to CEDAR
+func NewValidateAndSubmitSystemIntake(
+	validateAndSubmit func(intake *models.SystemIntake, logger *zap.Logger) (string, error),
+) func(intake *models.SystemIntake, logger *zap.Logger) (string, error) {
+	return func(intake *models.SystemIntake, logger *zap.Logger) (string, error) {
+		if intake.AlfabetID.Valid {
```

Comment:
Is this repeated code from the other service function?

Reply:
yep, I was planning on moving it, but forgot to remove it from the save system intake part.